tests/models/bart/test_modeling_flax_bart.py

```diff
@@ -426,7 +426,7 @@ def test_summarization_fast(self):
         )
 
         input_ids = tokenizer(input_str, return_tensors="np").input_ids
-        sequences = model.generate(input_ids, num_beams=2, max_length=20).sequences
+        sequences = model.generate(input_ids, num_beams=2, min_length=None, max_length=20).sequences
```

Comment:
(FLAX test failing in `main` as well)
this model's generation_config.min_length is 56, which clashes with the purpose of the test (being fast)